packages/buckram/assets/styles/variables/_specials.scss

```diff
@@ -964,16 +964,54 @@ $examples-background: #f3e1e3 !default;
 /// @type String
 $examples-header-background: #7a333a !default;
 
+// Contributors
+
+/// Margin top for elements with a class of `contributors`.
+/// @type String
+$contributors-margin-top: 2rem !default;
+
+/// Margin bottom for elements with a class of `contributors`.
+/// @type String
+$contributors-margin-bottom: 2rem !default;
```

Comment:
Should we simply create a single $contributors-margin variable or is it better to define them separately?